### DISCUSSION TITLE
How to get maximum disk capacity? What Erasure Coding should I set? (or maybe different settings?) I want to get maximum capacity from 4 disks.

### DISCUSSION THREAD

**Discussion by martinenkoEduard** [minio/minio#16807] [Q&A] [answered]
How to get maximum disk capacity? What Erasure Coding should I set? (or maybe different settings?) I want to get maximum capacity from 4 disks.

**Comment by vadmeste** [comment, marked as answer]
@martinenkoEduard you can use this link, https://min.io/product/erasure-code-calculator to calculate the usable capacity.

**Reply by martinenkoEduard** [reply]
I tried but i get only 50%  of usable capacity when using 4 disks.
Is there a way to increase this number?
